src/main/java/pages/BasePage.java

```diff
@@ -9,22 +10,28 @@
 public class BasePage {
 
     protected WebDriver driver;
+    protected Logger log;
 
     private By headerSignInLink = By.className("login");
     private By headerSignOutLink = By.className("logout");
     private By headerUsernameLink = By.className("account");
     private By headerContactUsLink = By.id("contact-link");
     private By headerLogo = By.id("header_logo");
+    private By pageH1 = By.className("page-heading");
 
-    public BasePage(WebDriver driver) {
+    public BasePage(WebDriver driver, Logger log) {
```

Comment:
I'm not sure why it was necessary to pass the Logger to constructor; for me it seems that it isn't necessary.